src/interface/storage.thrift

```diff
@@ -199,6 +207,8 @@ struct AddEdgesRequest {
     2: map<common.PartitionID, list<Edge>>(cpp.template = "std::unordered_map") parts,
     // If true, it equals an upsert operation.
     3: bool overwritable,
+    // index item
+    4: optional list<IndexItem> indexes,
```

Comment:
Why we need the indexes here?  
Storaged could read the information from metad.  We don't need to pass the information every request.

Reply:
> Why we need the indexes here?
> Storaged could read the information from metad. We don't need to pass the information every request.

Aha, Index meta wasn't available then , let me improve it now.